src/client/test/components/matchResults/index.test.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { render, fireEvent, cleanup } from '@testing-library/react';
+
+import MatchResults from 'client/components/matchResults';
+import { titles, buttonText } from 'client/constants/base';
+
+describe('<MatchResults />', () => {
+  afterEach(cleanup);
+
+  describe('render', () => {
+    it('should render correctly', () => {
+      const { getByText, queryByText } = render(<MatchResults />);
+
+      expect(queryByText(titles.matchResult)).not.toBeNull();
+      expect(queryByText(titles.scoreKeepByInning)).toBeNull();
+
+      fireEvent.click(getByText(buttonText.scoreKeepByInning));
```

Comment:
This test is a little bit weird to me 🤔 
IIUC, the `<MatchResults />` is kind of your `<App />` which I mean the root component to render the page so we should have tested other components.

Or, we should say the test here is to test if the click function works correctly instead of `render correctly`.

Also, maybe it's even better to test this kind of things via E2E tests? 🤔

Maybe I'm wrong; I'm not so good at testing. I will also pass this question to @Chun-MingChen, the KING of unit tests.

Reply:
Hmm, I agree I might have wrong description.
I'm still trying to know how to use `@testing-library` correctly, I'll keep on investigating.

Reply:
> trying to know how to use `@testing-library` correctly

From the point in the https://github.com/ss77995ss/sean-baseball-statistics/pull/3#discussion_r326933552 comment, it's about how to test instead of how to use which framework or library ;)

Reply:
You're right I need to ensure what to test and how to test first,  but I think I may also need to get more familiar with the library so that can design more precise testing ><.
BTW, thanks for the advice :)